src/main/java/mate/academy/library/dto/BookSearchParametersDto.java

```diff
@@ -0,0 +1,5 @@
+package mate.academy.library.dto;
+
+public record BookSearchParametersDto(String[] titles, String[] authors) {

```

Comment:
it's a better practice to either make all your DTOs as records, or make all of them as simple classes, mix of both can be confusing